### PR TITLE
Fixing install.sh one-liner

### PR DESCRIPTION
**Pull request by mdrohmann**
https://www.pivotaltracker.com/story/show/175722288

### CODE REVIEW

**Review by MDrakos — APPROVED** [review]
I've made this mistake as well. That's a tricky variable name. Might be worth a rename or adding an underscore. I'll leave it up to you.

**Comment by mdrohmann** [comment]
I decided to merge right away, as some users are running into that already...  But it is a good idea to add an underscore.

**Pull request merged**
